daemon/web/styles/typography.sass

```diff
@@ -0,0 +1,50 @@
+/* Text styles */
+
+=Paragraph()
```

Comment:
didn't know this was valid sass syntax 🤔

Reply:
@asitu could you provide a link, or explain what this syntax does? I've been trying to Google it, but there's nothing out there.